Header.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { View, Text, StyleSheet, Button } from "react-native";
+
+export const Headers = () => (
+  <View style={styles.container}>
+    <Text style={styles.left}>Cancel</Text>
+    <Text style={styles.main}> New task </Text>
+    <Text style={styles.right}>Save</Text>
+  </View>
+);
+
+const styles = StyleSheet.create({
+  container: {
+    height: 70,
+    paddingTop: 30,
+    backgroundColor: "rgb(31, 134, 255)",
+    display: "flex",
```

Comment:
You don't have to specify `display` since React Native defaults to `flex`. Remember that RN doesn't work like the web where you have several `display` options. Here you just have: `display: "flex"` and `display: "none"` (when you try to hide sth).

Check out the `display` section for more info: https://reactnative.dev/docs/layout-props#display
I encourage you to also take a look at some of the other layout/style props 💪